packages/core-cordova/www/metrics.js

```diff
@@ -0,0 +1,57 @@
+const MetricsService = cordova.require("core.core").MetricsService;
+
+const MOBILE_CORE_CLASS = "MobileCoreModule";
+
+class MetricsImpl {
+
+  constructor(id, data) {
+    this.id = id;
+    this.data = data;
+  }
+
+  collect() {
+    return this.data;
+  }
+}
+
+class CordovaMetricsService extends MetricsService {
```

Comment:
[Optional] Let's add documentation explaining what is happening here.

Reply:
Is it too much now maybe?